package.json

```diff
@@ -127,5 +127,8 @@
 				]
 			]
 		}
+	},
+	"dependencies": {
+		"events": "^3.3.0"
```

Comment:
I don't seem to understand why you list events as a dependency, it's a built in package for node.

the NPM page itself says:
> You usually do not have to install events yourself! If your code runs in Node.js, events is built in. If your code runs in the browser, bundlers like browserify or webpack also include the events module.

and browser isn't supported anymore so I don't see why this is here.

Reply:
Will fix